wgs.yaml

```diff
@@ -56,6 +56,67 @@ workinggroups:
   - name: katib
     owners:
     - https://raw.githubusercontent.com/kubeflow/katib/master/OWNERS
+- dir: wg-deployment
+  name: Deployment
+  mission_statement: >
+    WG Deployment covers developing, maintaining artifacts to deploy and operate Kubeflow.
+
+  charter_link: charter.md
+  label: area/wg-deployment
+  leadership:
+    chairs:
+    - github: Jeffwan
```

Comment:
Should we make it right order ?

Reply:
I think the tooling kind of orders by company